lib/mix/tasks/torch.gen.html.ex

```diff
@@ -19,7 +19,7 @@ defmodule Mix.Tasks.Torch.Gen.Html do
 
     %{format: format} = Mix.Torch.parse_config!("torch.gen.html", args)
 
-    Mix.Torch.ensure_phoenix_is_loaded("torch.install")
+    Mix.Torch.ensure_phoenix_is_loaded("torch.gen.html")
```

Comment:
This commit should be squashed back into the commit where the typo was originally introduced here: f9f9ee1